src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -482,13 +486,18 @@ export default defineComponent({
       return this.isInQuickBookmarkPlaylist ? 'base favorite' : 'base'
     },
 
-    watchPageLinkTo() {
-      // For `router-link` attribute `to`
+    watchVideoRoute() {
```

Comment:
Lets combine these two computed properties into a single one, as they are quite simple.
```suggestion
    watchVideoRouterLink() {
```